pkg/storage/rangefeed/resolved_timestamp.go

```diff
@@ -414,6 +462,24 @@ func (uiq *unresolvedIntentQueue) updateTxn(
 	return false
 }
 
+// Del removes the transaction from the queue. It returns whether the update had
```

Comment:
This method is exactly what was removed in https://github.com/cockroachdb/cockroach/pull/35777/files#diff-425bf0ccd6243bd7bb88910fc308bdf4L424.